docs/training/working-with-data-and-model-artifacts.md

```diff
@@ -0,0 +1,87 @@
+(working-with-data-and-model-artifacts)=
+# Working with Data and Model Artifacts
+
+When running a training job, you will need to pass in data to train and save the resulting model. Both the data and model can be considered [Artifacts](https://docs.mlrun.org/en/latest/store/artifacts.html) in MLRun. In the context of an ML pipeline, the data would be an `input` and the model would be an `output`.
+
+Consider the following snippet from a pipeline in the [Build and run automated ML pipelines and CI/CD](https://docs.mlrun.org/en/latest/tutorial/04-pipeline.html#build-and-run-automated-ml-pipelines-and-ci-cd) section of the docs:
+
+```python
+# Ingest data
+...
+
+# Train a model using the auto_trainer hub function
+train = mlrun.run_function(
+    "hub://auto_trainer",
+    inputs={"dataset": ingest.outputs["dataset"]},
+    params = {
+        "model_class": "sklearn.ensemble.RandomForestClassifier",
+        "train_test_split_size": 0.2,
+        "label_columns": "label",
+        "model_name": 'cancer',
+    }, 
+    handler='train',
+    outputs=["model"],
+)
+
+### Deploy model
+...
+```
+
+This snippet will train a model using the data provided into `inputs` and pass the model to the rest of the pipeline using the `outputs`.
+
+### Input Data
+
+The `inputs` parameter is a dictionary of key-value mappings. In this case, our input is our `dataset` (which is actually an output from a previous step). Within our training job, we can access the `dataset` input as an MLRun [DataItem](https://docs.mlrun.org/en/latest/concepts/data-items.html) (essentially a smart data pointer that provides convenience methods).
```

Comment:
The `inputs` parameter is a dictionary of key-value mappings. In this case, the input is the `dataset` (which is actually an output from a previous step). Within the training job, you can access the `dataset` input as an MLRun [Data Item](../concepts/data-items.html) (essentially a smart data pointer that provides convenience methods).